integration-cli/docker_cli_rename_test.go

```diff
@@ -39,8 +45,7 @@ func (s *DockerSuite) TestRenameRunningContainer(c *check.C) {
 }
 
 func (s *DockerSuite) TestRenameRunningContainerAndReuse(c *check.C) {
-	testRequires(c, DaemonIsLinux)
-	out, _ := dockerCmd(c, "run", "--name", "first_name", "-d", "busybox", "top")
+	out, _ := dockerCmd(c, "run", "--name", "first_name", "-d", "busybox", sleepCmd, "60")
```

Comment:
Yes... but unfortunately the version of busybox we're using is GPL, and our legal dept won't let us submit code to it without hoops you would not believe.....